demo/index.html

```diff
@@ -30,87 +30,92 @@
-      color: var(--paper-grey-700);
-    }
-
-    .set:nth-of-type(4n-2) {
-      color: var(--paper-pink-500);
-    }
-
-    .set:nth-of-type(4n-1) {
-      color: var(--google-green-500);
-    }
-
-    .set:nth-of-type(4n) {
-      color: var( --google-blue-500);
-    }
-
-    .container {
-      min-width: 10em;
-      padding: 1em 0.5em;
-      text-align: center;
-      @apply(--layout-vertical);
-      @apply(--layout-center);
-      @apply(--layout-flex);
-    }
-
-    .container > div {
-      margin-top: 0.5em;
-      color: black;
-      font-size: 10px;
-    }
-  </style>
 </head>
 <body>
-  <template is="dom-bind">
-   <iron-meta type="iconset" list="{{iconsets}}"></iron-meta>
-
-    <template is="dom-repeat" items="{{iconsets}}">
-      <h2>{{item.name}}</h2>
-
-      <div class="set">
-        <template is="dom-repeat" items="{{getIconNames(item)}}">
-          <span class="container">
-            <iron-icon icon="{{item}}"></iron-icon>
-            <div>{{item}}</div>
-          </span>
+  <custom-style>
+    <style is="custom-style">
+      h2 {
+        text-transform: capitalize;
+      }
+
+      iron-icon {
+        transition: all 0.2s;
+        -webkit-transition: all 0.2s;
+      }
+
+      iron-icon:hover {
+        fill: var(--google-yellow-700);
+      }
+
+      .set {
+        display: block;
+        text-align: center;
+        margin: auto;
+        padding: 1em 0;
+        border-bottom: 1px solid silver;
+      }
+
+      .set:last-of-type {
+        border-bottom: none;
+      }
+
+      .set:nth-of-type(4n-3) {
+        color: var(--paper-grey-700);
+      }
+
+      .set:nth-of-type(4n-2) {
+        color: var(--paper-pink-500);
+      }
+
+      .set:nth-of-type(4n-1) {
+        color: var(--google-green-500);
+      }
+
+      .set:nth-of-type(4n) {
+        color: var(--google-blue-500);
+      }
+
+      .container {
+        display: inline-block;
+        width: 10em;
+        margin: 1em 0.5em;
+        text-align: center;
+      }
+
+      .container > div {
+        margin-top: 0.5em;
+        color: black;
+        font-size: 10px;
+      }
+    </style>
+  </custom-style>
+
+  <dom-bind>
+    <template is="dom-bind" id="scope">
+      <dom-repeat>
+        <template is="dom-repeat" id="iconsetRepeat">
+          <h2>{{item.name}}</h2>
+          <dom-repeat class="set">
+            <template is="dom-repeat" items="{{getIconNames(item)}}">
+              <span class="container">
+                <iron-icon icon="{{item}}"></iron-icon>
+                <div>{{item}}</div>
+              </span>
+            </template>
+          </dom-repeat>
         </template>
-      </div>
+      </dom-repeat>
     </template>
-  </template>
+  </dom-bind>
 
   <script>
-    document.querySelector('[is=dom-bind]').getIconNames = function(iconset) {
-      return iconset.getIconNames();
-    };
+    window.addEventListener('WebComponentsReady', function() {
+      scope.getIconNames =
```

Comment:
this trailing `=` is intentional, right? i might move the `scope.parentNode` line below to make that explicit, or indent it by 4. or something. right now it loos like a typo :)